tests/inc/test-update.php

```diff
@@ -318,6 +318,11 @@ function test_largo_transition_nav_menus() {
 	}
 
 	function test_largo_update_prominence_term_description_single() {
+		$term9 = get_term_by('slug', 'term-9', 'prominence', 'ARRAY_A');
+		if ( ! $term9 ) {
+			wp_insert_term( 'term-9', 'prominence', array( 'description' => 'Term Description 9' ) );
```

Comment:
Previous comment might be wrong, based on https://github.com/INN/Largo/issues/1209#issuecomment-225950908

Either way, the test tests what it's supposed to test.